exec/java-exec/src/test/resources/store/text/WithQuotedCrLf.tbl

```diff
@@ -0,0 +1,6 @@
+"a
+1"|a|a
+a|"a
+2"|a
+a|a|"a
+3"
```

Comment:
Is there an issue with git converting Windows-style newlines (\r\n) into Unix-style (\n) when this file is checked in & out? Will that mess up the test? Should the test generate this file to handle this particular special case?